vowpalwabbit/conditional_contextual_bandit.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+#include "label_parser.h"
+#include "v_array.h"
+#include "action_score.h"
+
+struct vw;
+struct example;
+typedef std::vector<example*> multi_ex;
+
+namespace CCB {
+  // Each positon in outer array is implicitly the decision corresponding to that index. Each inner array is the result of CB for that call.
+  typedef v_array<ACTION_SCORE::action_scores> decision_scores_t;
+
+  struct conditional_contexual_bandit_outcome

```

Comment:
rename `conditional_contexual_bandit_outcome` in `ccb_outcome`?

Reply:
I opted for the long form throughout as ccb is easy to mixup with cb. Thoughts?

Reply:
I am fine with both, so let's do what you prefer.
I prefer `ccb_outcome` regarding cosmetics mainly. Also because i prefer shorter names.